client/src/lib/bowling.js

```diff
@@ -0,0 +1,71 @@
+class Bowling {
+    constructor() {
+        this.rolls = [];
+        this.score = 0;
+        this.frame = 0;
+        this.rollsIndexPosition = 0
+    };
+
+    getScore() {
+        if(this.rolls.length > 0) {
+            this.calculateScore()
+        }
+        return this.score;
+    }
+
+    roll(score) {
+        if(score < 0 || score > 10) {
+            throw "roll must be between 0 and 10"
+        }
+        this.rolls.push(parseInt(score));
+    }
+
+    isAStrike() {
+        return this.rolls[this.rollsIndexPosition] === 10;
+    }
+
+    isASpare() {
+        return (this.rolls[this.rollsIndexPosition] + this.rolls[this.rollsIndexPosition + 1]) === 10;
+    }
+
+    isLastFrame() {
+        return this.frame === 10;
```

Comment:
I'm pretty sure that you are zero-indexing, so the last frame of 10 frames will be index 9. This broken logic would prompt another test to be written.